### DISCUSSION TITLE
feat: Automatically hide the mouse (cursor) after when not being moved after X seconds

### DISCUSSION THREAD

**Discussion by fooness** [ghostty-org/ghostty#2999] [Feature Requests, Ideas] [answered]
In addition to the `mouse-hide-while-typing` option, it would be great if the mouse cursor would be automatically hidden after some period of inactivity/resting. In other words: after moving the cursor within Ghostty, i.e. when having stopped moving the cursor, wait for example `5` seconds, then automatically hide the cursor.

**Comment by murlidhar** [comment]
there are options in other terminal emulators too for this already. ... eg mouse_hide_wait = n no of seconds. the mouse pointer will remain hidden until the mouse is moved.

**Comment by mitchellh** [comment, marked as answer]
#8384

**Comment by Mayurifag** [comment]
Just curious, what is practical use case? I wonder if that might be applicable to my workflow 😄
